.github/workflows/ci-pipeline.yml

```diff
@@ -89,6 +89,52 @@ jobs:
             } else {
               core.warning(message);
             }
+  check-images:
+    runs-on: ubuntu-latest
+    needs: [get-chart]
+    name: Look for hardcoded images
+    if: needs.get-chart.outputs.result == 'ok'
+    outputs:
+      result: ${{ steps.check-images.outputs.result }}
+      error: ${{ steps.check-images.outputs.error }}
+    steps:
+      - name: Checkout bitnami/charts
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
+        with:
+          ref: ${{github.event.pull_request.head.ref}}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
+          path: charts
+      - id: check-images
+        name: Look for hardcoded images
+        env:
+          CHART: ${{ needs.get-chart.outputs.chart }}
+        run: |
+          hardcoded_images=()
+          while read -r image; do
+            if [[ $image != {{*}} ]]; then
+              hardcoded_images+=("${image}")
+            fi
+          done <<< "$(grep -REoh "image:\s+[\"']*.+[\"']*\s*$" "charts/bitnami/${CHART}/templates" | sed "s/image: [\"']*//" | sed "s/[\"']*$//")"
```

Comment:
Should the 1st regexp be like this instead?
```diff
-image:\s+[\"']*.+[\"']*\s*$
+^\s*image:\s+[\"']*.+[\"']*\s*$
```
Regarding the last "sed", maybe it's simpler to use `tr -d '"'`

Reply:
I will add the `^\s*` to the regexp for extra protection in case of mismatch / false positive.

Regarding the `tr -d` I'm not so sure, since I want the command to remove both single and double quotes exclusively from the end of the string. For example, from `"{{ include "helper.example" . }}"` I want to keep inner quotes, not remove every quote.